src/app/signalen/page.tsx

```diff
@@ -19,29 +29,24 @@ function Contact() {
       <Grid.Cell span={{ narrow: 4, medium: 6, wide: 7 }} start={{ narrow: 1, medium: 2, wide: 2 }}>
         <form className="ams-gap--md" onSubmit={handleSubmit}>
           <Heading>Doe een melding</Heading>
-
           <Heading level={2}>Beschrijf uw melding</Heading>
-
-          <div>
+          <Field>
```

Comment:
The same comments as in [amsterdam/contact/vraag/page.tsx](https://github.com/Amsterdam/design-system-prototypes/pull/168/files#diff-e901f5bb264d249d5690ce368ef191a6feb26cea5e7b4d7b6c5fa7da3c45ce88) apply here, because they're pretty much the same page. Why do we have 2 of these pages?

Reply:
Because we offer both sites as examples?

Reply:
Ah, I didn't know amsterdam.nl sort of remade meldingen.amsterdam.nl in TripleForms. Feels a little pointless imo, but it makes sense to include the current situation in our prototypes.